package.json

```diff
@@ -46,8 +46,11 @@
     "ng2-pagination": "^2.0.1"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "2.1.3",
-    "@types/jasmine": "^2.5.41",
+    "@ionic/app-scripts": "^1.3.7",
```

Comment:
It looks like you downgraded a bit here.  You should update your copy of Ionic to the [recommended version here](https://github.com/umts/pvta-multiplatform#prerequisites).

Reply:
The comment I left on #434 applies here verbatim.